mirrormanager2/templates/fedora/host.html

```diff
@@ -1,267 +1,246 @@
-  <li>
-      <form class="icon" method="POST"
-        action="{{ url_for('base.host_acl_ip_delete', host_id=host.id, host_acl_ip_id=acl.id) }}">
-      {{ acl.ip }}
-      {{ form.csrf_token }}
-        <button
-          onclick="return confirm('You sure you want to delete this ACL?');"
-          title="Delete ACL IP">
-          <span class="icon icon-trash blue"></span>
-        </button>
-      </form>
-    </li>
-  {% endfor %}
-  </ul>
-{% endif %}
-{%- endif -%}{# MASTER_RSYNC_ACL ##}
-
-<h3>
-  Site-local Netblocks
-</h3>
-<p>
-  Netblocks are used to try to guide and end user to a site-specific mirror.
-  For example, a university might list their netblocks, and the mirrorlist
-  CGI would return the university-local mirror rather than a country-local
-  mirror. Format is one of 18.0.0.0/255.0.0.0, 18.0.0.0/8, an IPv6
-  prefix/length, or a DNS hostname. Values must be public IP addresses (no
-  RFC1918 private space addresses).
-</p>
-
-<a class="icon icon-plus-sign blue"
-  href="{{ url_for('base.host_netblock_new', host_id=host.id) }}"> [add]</a>
-{% if host.netblocks %}
-  <ul>
-  {% for netblock in host.netblocks %}
-    <li>
-      <form class="icon" method="POST"
-        action="{{ url_for('base.host_netblock_delete', host_id=host.id,
-                   host_netblock_id=netblock.id) }}">
-      {{ netblock.netblock }}
-      {{ form.csrf_token }}
-        <button
-          onclick="return confirm('You sure you want to delete this netblock?');"
-          title="Delete netblock">
-          <span class="icon icon-trash blue"></span>
-        </button>
-      </form>
-    </li>
-  {% endfor %}
-  </ul>
-{% endif %}
-
-<h3>
-  Peer ASNs
-</h3>
-<p>
-  Peer ASNs are used to guide an end user on nearby networks to our mirror.
-  For example, a university might list their peer ASNs, and the mirrorlist
-  CGI would return the university-local mirror rather than a country-local
-  mirror. You must be in the MirrorManager administrators group in order to
-  create new entries here.
-</p>
-
-{% if is_admin %}
-<a class="icon icon-plus-sign blue"
-  href="{{ url_for('base.host_asn_new', host_id=host.id) }}"> [add]</a>
-{% endif %}
-{% if host.peer_asns %}
-  <ul>
-  {% for asn in host.peer_asns %}
-    <li>
-    {% if is_admin %}
-      <form class="icon" method="POST"
-        action="{{ url_for('base.host_asn_delete', host_id=host.id,
-                   host_asn_id=asn.id) }}">
-        {{ asn.asn }}
-        {{ form.csrf_token }}
-        <button
-          onclick="return confirm('You sure you want to delete this Peer ASN?');"
-          title="Delete Peer ASN">
-          <span class="icon icon-trash blue"></span>
-        </button>
-      </form>
-    {% else %}
-      {{ asn.asn }}
+    {% if not host.private %}
+    <div><strong>Last Crawled:</strong> {% if host.last_crawled %} {{ host.last_crawled
+      }} <a href="{{url_for('base.crawler_log', host_id=host.id)}}">[Log]</a> {% else %} - {% endif %} </div>
```

Comment:
Maybe "never" rather than "-" here?

Reply:
yeah -- it was always '-' but never is better. fixed.